### PR TITLE
dx-728 Added new screenshots

### PR DESCRIPTION
**Pull request by arebra**
Added new payment menu screenshots

### CODE REVIEW

File: checkout/payment-menu.md [hunk 1]
```diff
@@ -301,7 +307,7 @@ finalizing the payment in the [After Payment section][after-payment].
 [operations]: /checkout/other-features#operations
 [order-items]: #order-items
 [payee-reference]: /checkout/other-features#payeereference
-[payment-menu-image]: /assets/img/checkout/payment-menu.png
+[login-payment-menu-image]: /assets/img/checkout/LoggedInPaymentMenu450x900.png
```
Comment (asbjornu) [inline checkout/payment-menu.md]:
```suggestion
[login-payment-menu-image]: /assets/img/checkout/logged-in-payment-menu-450x900.png
```

File: checkout/payment-menu.md [hunk 2]
```diff
@@ -293,6 +298,7 @@ finalizing the payment in the [After Payment section][after-payment].
                          next_href="capture"
                          next_title="Next: Capture" %}
 
+[anon-payment-menu-image]: /assets/img/checkout/Newpaymentmenu450x850.png
```
Comment (asbjornu) [inline checkout/payment-menu.md]:
```suggestion
[guest-payment-menu-image]: /assets/img/checkout/guest-payment-menu-450x850.png
```

File: checkout/payment-menu.md [hunk 3]
```diff
@@ -265,10 +265,15 @@ request.send(JSON.stringify({
...
+
+This should bring up the Payment Menu in a Seamless View. It should look like
+this, depending on wether the payer is logged in (top) or a guest user (bottom):
+
+{:.text-center}
+![Payment Menu with payer logged in and card payment opened][login-payment-menu-image]{:width="450" height="900"}
 
 {:.text-center}
-![Payment Menu with card payment opened][payment-menu-image]{:width="407" height="627"}
+![Payment Menu with guest payer and card payment opened][anon-payment-menu-image]{:width="450" height="850"}
```
Comment (asbjornu) [inline checkout/payment-menu.md]:
```suggestion
![Payment Menu with guest payer and card payment opened][guest-payment-menu-image]{:width="450" height="850"}
```

File: checkout/payment-menu.md [hunk 4]
```diff
@@ -265,10 +265,15 @@ request.send(JSON.stringify({
 }));
 ```
 
-This should bring up the Payment Menu in a Seamless View, looking like this:
+
+This should bring up the Payment Menu in a Seamless View. It should look like
+this, depending on wether the payer is logged in (top) or a guest user (bottom):
```
Comment (trieutran91) [inline checkout/payment-menu.md]:
```suggestion
this, depending on whether the payer is logged in (top) or a guest user (bottom):
```

File: checkout/payment-menu.md [hunk 5]
```diff
@@ -301,7 +307,7 @@ finalizing the payment in the [After Payment section][after-payment].
 [operations]: /checkout/other-features#operations
 [order-items]: #order-items
 [payee-reference]: /checkout/other-features#payeereference
-[payment-menu-image]: /assets/img/checkout/payment-menu.png
+[login-payment-menu-image]: /assets/img/checkout/logged-in-payment-menu-450x900.png
```
Comment (trieutran91) [inline checkout/payment-menu.md]: Tip: Use command palette (Ctrl+Shift+P) and the "Sort Lines Ascending" to sort the links alphabetically.

**Pull request merged**